app/controllers/assessment/autograde.rb

```diff
@@ -1,23 +1,18 @@
 module AssessmentAutograde
   require 'autoConfig.rb'
 
-  # 
-  # autogradeAfterHandin - submits an autograding job to Tango when
-  # the students submit their work.
-  #
-  def autogradeAfterHandin(submission)
+  ##
+  # autogradeSubmissions - submits an autograding job to Tango.
+  # Called by assessments#handin, submissions#regrade and submissions#regradeAll
+  # returns the job status returned by sendJob
+  def autogradeSubmissions(course, assessment, submissions)
```

Comment:
I don't understand. Why are we passing multiple submissions here but only using submissions[0] throughout?